ui_vue/src/App.vue

```diff
@@ -44,14 +47,54 @@ watch(route, () => {
 <template>
   <div>
     <q-layout view="hHh Lpr lff">
+      <TheNavBar v-if="showNavbar" />
       <q-page-container>
-        <q-page>
+        <q-page class="overflowX-hidden">
+          <div
+            class="row justify-between items-center q-ma-lg q-pa-md borderRadius-1rem bg-v-light-gray"
+            v-if="showNavbar"
+          >
+            <div class="row items-center gap-half-rem">
+              <q-icon name="home_app_logo" class="icon-size" color="positive" />
+              <p class="m-0 route-size font-size">{{ route.name }}</p>
+            </div>
+            <p class="m-0 text-grey font-size">{{ route.path }}</p>
+          </div>
           <RouterView />
           <q-page-scroller position="bottom-right" :scroll-offset="150" :offset="[18, 18]">
-            <q-btn fab icon="keyboard_arrow_up" color="accent" />
+            <q-btn fab icon="keyboard_arrow_up" color="accent" padding="md"/>
           </q-page-scroller>
         </q-page>
       </q-page-container>
     </q-layout>
   </div>
 </template>
+
+<style scoped lang="scss">
+@import './styles/breakpoints.scss';
```

Comment:
Why the styles are not in a separate file?

Reply:
moved to the dedicated styling files